control/DeadbandXboxController.java

```diff
@@ -19,6 +39,23 @@
     public DeadbandXboxController(int port, double deadbandCutoff) {
         super(port);
         this.deadbandCutoff = deadbandCutoff;
+
+        aButton = new JoystickButton(this, DeadbandXboxController.Button.kA.value);
+        bButton = new JoystickButton(this, DeadbandXboxController.Button.kB.value);
+        xButton = new JoystickButton(this, DeadbandXboxController.Button.kX.value);
+        yButton = new JoystickButton(this, DeadbandXboxController.Button.kY.value);
+        leftBumper = new JoystickButton(this, DeadbandXboxController.Button.kLeftBumper.value);
+        rightBumper = new JoystickButton(this, DeadbandXboxController.Button.kRightBumper.value);
+        dPadUp = new POVButton(this, 0);
+        dPadDown = new POVButton(this, 180);
+        dPadLeft = new POVButton(this, 270);
+        dPadRight = new POVButton(this, 90);
+        leftXTrigger = new Trigger(() -> {return this.getLeftX() != 0;});
```

Comment:
These 4 are the joysticks on the XboxController, can we rename them to like leftStickX, rightStickY, etc.?

Reply:
I'm trying to be more descriptive as those names generally just describe the axis value associated with the stick rather than a trigger assigned to the joystick

Reply:
Like when we're trying to implement it in another project it can easily be confused with an actual Xbox trigger.